sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/CreateBatchOptions.cs

```diff
@@ -106,15 +54,6 @@ public class CreateBatchOptions
                 _maximumSizeInBytes = MaximumSizeInBytes
             };
 
```

Comment:
Ah....I don't think anyone will miss this convention :)

Also, @JoshLove-msft , look - we had a 'ToSomething' converter method!

Reply:
The only reason that we removed the inheritance relationship that I just restored was because we internalized the send options for a while _(and I was too lazy to implement implicit conversions)_